blueprints/hpc-cluster-small.yaml

```diff
@@ -28,6 +28,17 @@ deployment_groups:
         id: network1
         use: [project1]
 
+      - source: ./modules/billing
+        kind: terraform
+        id: billing
+        settings:
+          display_name: example-alert"
```

Comment:
1. Is this `display_name` required? If not, maybe it can be removed?
2. In case the `display_name` will remain, could you add a identifiers to it (project Id for example)?